multi_lidar_calibrator/multi_lidar_calibrator.py

```diff
@@ -504,10 +494,13 @@ def pointcloud_callback(self, msg: PointCloud2):
                 return  # Don't save more point clouds than needed
             self.lidar_data[msg.header.frame_id].append(msg)
```

Comment:
Can you please change this logic to always have the latest 5 frames? Kinda like a circular buffer